cmd/juju/firewall/listrules_test.go

```diff
@@ -91,17 +98,20 @@ func (s *ListSuite) assertValidList(c *gc.C, args []string, expectedValid, expec
 }
 
 type mockListAPI struct {
-	rules []params.FirewallRule
+	rules string
 	err   error
 }
 
 func (s *mockListAPI) Close() error {
 	return nil
 }
 
-func (s *mockListAPI) ListFirewallRules() ([]params.FirewallRule, error) {
+func (s *mockListAPI) ModelGet() (map[string]interface{}, error) {
 	if s.err != nil {
 		return nil, s.err
 	}
-	return s.rules, nil
+	return testing.FakeConfig().Merge(testing.Attrs{
+		config.SSHAllowKey:         s.rules,
+		config.SaasIngressAllowKey: "0.0.0.0/0",
```

Comment:
Again, acronyms should be all capitals in go (see SSH as an example in the line above).